lib/diagnostic.rb

```diff
@@ -74,17 +85,21 @@ def initialize
 
 ## What will be the output from calling `HouseCat.new.say_hello`?
 # replace nil with your response
-Response.housecat_noise = nil
+Response.housecat_noise = "'lick... lick...''I am a animal and I go 'meow'"
 
 ## Explain why this would be the output, based on the method lookup chain.
 # your response as a comment here
 ##
-
+Calling HouseCat, initilizes @sound as 'meow', goes up to Cat, puts 'lick...lick'
+which is an instance method of Class Animal, which calls say_hello, which puts
+"I am a #{self.class.name}, and I go '#{@sound}'", self refers to Class Animal, and @sound
+refers to an initialized sound, which was defined with HouseCat.
```

Comment:
remember that whenever an object's properties or methods are called, it first looks at it's own class, and only when it doesn't find it, does it traverse up the inheritance tree